Song_sort_Genre.php

```diff
@@ -1,39 +1,70 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title>Layout1Musicplay</title>
+		<title>Sort By Genre</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
+		<!--Connected to css -->
 		<link rel ="stylesheet" href = "test12.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
-		<div class="img_1">
-			<img src = "images/img.png">
-		</div>
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
-			<nav>
```

Comment:
21/9: I delete the nav bar to make it better by make a function that called up the nav so I will delete this nav. Also this nav wasn't accurate. And I think if I could create a page for nav and latter on I just need to called it then it would be easy because if I want to change something for the navigation I could just change in the nav page and it will automatic change for other page too.